.github/workflows/ci.yml

```diff
@@ -1026,12 +1026,13 @@ jobs:
       - name: Run Fluvio CLI smoke tests
         if: matrix.test == 'fluvio'
         uses: nick-fields/retry@v2
+        env:
+          FLUVIO_CLUSTER_RELEASE_CHANNEL: ${{ matrix.cluster_version }}
```

Comment:
This is not needed. There are already some lines above with:
```
echo "FLUVIO_CLUSTER_RELEASE_CHANNEL=${FLUVIO_CLUSTER_RELEASE_CHANNEL}"   >> $GITHUB_ENV
```

Reply:
Gotcha!